docs/documentation/server_admin/topics/authentication/webauthn.adoc

```diff
@@ -77,7 +76,6 @@ The user can choose between using WebAuthn and OTP for the second factor:
 . On the *Conditional 2FA* row, click the plus sign + and select *Add step*.
 . Click *Add step*.
 . Select *OTP Form* from the list.
-. Click *Add*.
```

Comment:
This step is perfectly fine and should stay. Step 2 on line 77 is supposed to be the one deleted